AlphaWallet/Tokens/ViewModels/EthCurrencyHelper.swift

```diff
@@ -24,6 +24,14 @@ class EthCurrencyHelper {
         }
     }
 
+    var marketPrice: Double? {
+        if let price = ticker?.price_usd {
+            return price.rounded(to: 2)
```

Comment:
Let's wait until #1970 is merged and then we can use CurrencyFormatter here. and a few lines below without the hardcoded '2'.

Reply:
@vladyslav-iosdev would you help to update and remove the '2' here and below since #1970 has been merged.